tests/symbolics/test_sympy_interpreter.py

```diff
@@ -397,3 +397,20 @@ def test_sympy_interpreter_warns_about_using_caret_sign_for_exponentiation():
     expr = "x ^ 2"
     with pytest.warns(match=r"Using \^ operator to denote exponentiation is deprecated\."):
         _ = parse_to_sympy(expr)
+
+
+@pytest.mark.parametrize(
+    "value,expected,raises,match",
+    [
+        (8, 3, None, None),  # integer input, should succeed
+        (8.0, None, TypeError, "must be an integer, not a float"),  # float integer, should raise
```

Comment:
I think if we implement the suggestion I made above we could just get rid of `match`, WDYT?

Reply:
thanks for reviewing, I just update it with the idea, take a look